atomate/qchem/database.py

```diff
@@ -0,0 +1,57 @@
+# coding: utf-8
+
+from __future__ import division, print_function, unicode_literals, absolute_import
+
+from monty.json import MontyEncoder
+"""
+This module defines the database classes.
+"""
+
+import zlib
+import json
+from bson import ObjectId
+
+import gridfs
+from pymongo import ASCENDING, DESCENDING
+
+from atomate.utils.database import CalcDb
+from atomate.utils.utils import get_logger
+
+__author__ = "Brandon Wood"
+__credits__ = "Kiran Mathew, Anubhav Jain"
+__email__ = "b.wood@berkeley.edu"
+
+logger = get_logger(__name__)
+
+
+class QChemCalcDb(CalcDb):
+    """
+    Class to help manage database insertions of QChem drones
+    """
+
+    def __init__(self,
+                 host="localhost",
+                 port=27017,
+                 database="qchem",
+                 collection="tasks",
+                 user=None,
+                 password=None):
+        super(QChemCalcDb, self).__init__(host, port, database, collection,
+                                          user, password)
+
+    def build_indexes(self, indexes=None, background=True):
+        """
+        Build the indexes.
+
+        Args:
+            indexes (list): list of single field indexes to be built.
+            background (bool): Run in the background or not.
+
+        TODO: Figure out what this does
```

Comment:
Building indexes just makes certain queries faster, it is not required. But if there are certain keys that you want to query on frequently, it's better to add an index so that querying on them is fast.

So I'd suggest just adding a few indices on the common query keys

Reply:
Thanks, added a few single field indices. Left building compound indices as a TODO once we have a better idea of how we'll be using our database.